README.md

```diff
@@ -17,8 +17,6 @@ This Node-based rig uses a Gulp workflow configured for GitHub Pages publishing.
 All the necessary installations are handled in [icj-setting-up](https://github.com/utdata/icj-setting-up).
 
 To start a new project:
-- Download Node.js version `16.18.0`.
-  - I would recommend using [NVM](https://heynode.com/tutorial/install-nodejs-locally-nvm/) to manage your node versions, as they tend to be different in every project.
```

Comment:
This is redundant since it is in the `icj-setting-up` repository, so I removed it.